pulsar-broker/src/test/java/org/apache/pulsar/schema/compatibility/SchemaCompatibilityCheckTest.java

```diff
@@ -478,7 +483,56 @@ public void testProducerSendWithOldSchemaAndConsumerCanRead(SchemaCompatibilityS
 
         consumerOne.close();
         producerOne.close();
+    }
 
+    @Test
+    public void testSchemaLedgerAutoRelease() throws Exception {
+        String namespaceName = PUBLIC_TENANT + "/default";
+        String topicName = "persistent://" + namespaceName + "/tp";
+        admin.namespaces().createNamespace(namespaceName, Sets.newHashSet(CLUSTER_NAME));
+        admin.namespaces().setSchemaCompatibilityStrategy(namespaceName, SchemaCompatibilityStrategy.ALWAYS_COMPATIBLE);
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        // Update schema 100 times.
+        ArrayList<Class> classes = createManyClass(classLoader, 100);
```

Comment:
I think using the SchemaDefinitionBuilder is easier to create multiple schemas.

Reply:
Already use `SchemaDefinitionBuilder` instead dynamic generate class